tsconfig.base.json

```diff
@@ -19,7 +19,7 @@
 		"noUncheckedIndexedAccess": true,
 		"typeRoots": ["./node_modules/@types", "./src"],
 		"types": ["node"],
-		"skipLibCheck": true,
+		"skipLibCheck": true, // disable this once our deps get their act together
```

Comment:
What's this for?

Reply:
`skipLibCheck` disables type checking on _every_ .d.ts file. With this disabled, our own .d.ts files get type checking, which is nice when I want static analysis on my `Command` and `CommandContext` family of interfaces. While developing these changes, there've been times when I messed up and accidentally ended up with implicit `any` types that ESLint couldn't catch, because I'd misspelled something in a .d.ts file. Disabling this caught those, and that's great.

But discord.js and undici, probably a few others, are deps or sub-deps of ours whose .d.ts files do _not_ pass type checks. I wish I knew a good way to tell TS to skip checking libs in node_modules, but the prevailing wisdom at the moment is that [the TypeScript ecosystem is young](https://stackoverflow.com/a/56268670), and hasn't figured out their crap yet, so meantime we should enable `skipLibCheck` if our deps are silly.

The note here is for me to periodically check if our deps have worked out their interfaces, maybe to make a PR on them once I get mad enough, and occasionally check to see if TS has added an option to disable node_modules typechecks.